### ISSUE TITLE
The Extensions From The Repo URL  Are Not Available 

### ISSUE THREAD

**Issue by Samdy1010** [keiyoushi/tachiyomi-extensions-source#3929]
### Describe why this should be added

Because after deleting my old extensions as instructed and adding the repo. After restarting Tachiyomi, the extension list was empty even after several attempts. 

### Other details

[
![Screenshot_20240709_171437_Tachiyomi](https://github.com/keiyoushi/extensions-source/assets/168037925/c0f4f371-93fd-431b-b8c5-58a2706afc7b)
![Screenshot_20240709_171443_Tachiyomi](https://github.com/keiyoushi/extensions-source/assets/168037925/52ca1709-52ba-447d-9441-134f9da30522)
](url)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/mihonapp/mihon/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Norsze** [comment]
We can't fix your internet connection to GitHub, which might be blocked by your ISP, with code in the repo. Come to the Discord if you want help with this
